translations/en/plugin-handbook.md

```diff
@@ -390,6 +396,11 @@ const MyVisitor = {
     console.log("Called!");
   }
 };
+
+// You can also create a visitor and add methods on it later
+let visitor = {};
+visitor.MemberExpression = function() {};
+visitor.FunctionDeclaration = function() {}
```

Comment:
Not necessary just writing how it's done (and yeah I guess it's obvious since it's an object)